src/amber/cli/templates/auth/src/views/layouts/+_nav.slang.ecr

```diff
@@ -1,8 +1,9 @@
-- if context.current_<%= @name %>
-  a class="nav-item pull-right" href="/signout" Sign Out
-  div.nav-item.pull-right == context.current_<%= @name %>.not_nil!.email
+- if (current_<%= @name %> = context.current_<%= @name %>)
+  a.nav-item.pull-right[href="/signout"] Sign Out
+  - active = context.request.path == "/profile" ? "active" : ""
+  a[class="nav-item #{active} pull-right" href="/profile"] == current_<%= @name %>.email
 - else
   - active = context.request.path == "/signup" ? "active" : ""
-  a class="nav-item #{active} pull-right" href="/signup" Sign Up
+  a[class="nav-item #{active} pull-right" href="/signup"] Sign Up
```

Comment:
Any thoughts on this syntax for `slang` I like it because it more explicitly separates attributes from content, but I could see the argument that it doesn't look as clean with the extra characters.

Reply:
because of the dynamic class of `active`, we cannot not use the more explicit class separator.

Reply:
Let me clarify

This (with square brackets):
```
a[class="nav-item #{active} pull-right" href="/signup"] Sign Up
```
vs this (without):
```
a class="nav-item #{active} pull-right" href="/signup" Sign Up
```

Reply:
ahhh, gotcha.  yes, go for it and remove them.

Reply:
I like the square brackets, maybe we put it to a vote.
👍  - for square brackets, 👎  - for white space

Reply:
Fine, we'll treat attributes and content the same, like barbarians 😉 
```slim
/ So clean and CSS-like
a[href="/signup"] Sign Up

/ But one less character tho
a href="/signup" Sign Up
```

Reply:
Well, we can use indentation too:

```slim
a href="/signup"
  | Sign Up
```